packages/utils/src/instrument/fetch.ts

```diff
@@ -205,12 +206,14 @@ export function parseFetchArgs(fetchArgs: unknown[]): { method: string; url: str
     return {
       url: getUrlFromResource(url),
       method: hasProp(options, 'method') ? String(options.method).toUpperCase() : 'GET',
+      body: hasProp(options, 'body') ? String(options.body) : null,
```

Comment:
According to the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Request/body), body can be null.

Reply:
Outdated,